receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -40,8 +40,9 @@ attributes:
 metrics:
   # CPU
   container.cpu.usage.system:
```

Comment:
All the cpu metrics seem to me inconsistent with specification and other cpu usage metrics. We better take the opportunity on fixing all of them now.

Reply:
@bogdandrutu can you clarify in what ways the CPU metrics are inconsistent with the spec?

The CPU metrics here are taken directly from what the Docker API provides - here is the [exact struct](https://github.com/moby/moby/blob/master/api/types/stats.go#L7-L55) we pull from.

The [semconv](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systemcpu---processor-metrics) doesn't define how to report cgroup/container level metrics. This receiver only reports cgroup/container metrics, with the exception of the system-level `container.cpu.usage.system` which comes from the Docker API. I believe its main purpose is to compare the system's overall usage with a container's usage, which can be done via `100*container.cpu.usage.total` / `container.cpu.usage.system` = container's CPU usage as a percentage of the system's. 

The semconv only defines system-level CPU metrics `system.cpu.time` and `system.cpu.utilization`. These don't align 100% with what the docker API provides. The only clash/duplicate is `container.cpu.usage.system` with the non-idle states of `system.cpu.time` - I'd be happy to remove `container.cpu.usage.system` from this receiver if you believe it is clashing with the semconv/spec/hostmetricsreceiver.

Reply:
@bogdandrutu any thoughts? I would like to get this PR moving.